packages/braid-design-system/src/lib/components/Pagination/Pagination.tsx

```diff
@@ -107,9 +109,10 @@ const Page = ({ number, current }: { number: number; current: boolean }) => {
         boxShadow="borderFormAccent"
         className={{
           [styles.lightModeCurrentKeyline]:
-            parentBackground.lightMode !== 'surface' && current,
-          [styles.darkModeCurrentKeyline]:
-            parentBackground.darkMode !== 'surface' && current,
+            isLegacyTheme &&
+            parentBackground.lightMode !== 'surface' &&
+            current,
+          [styles.darkModeCurrentKeyline]: current,
```

Comment:
Is [this statement on the `Pagination` docs](https://seek-oss.github.io/braid-design-system/components/Pagination/#contextual-design) still correct with this change?

Reply:
Both this change and the `Alert` change make that statement theme specific. Will have to consider how to approach this.